src/Components/RosTable/RenderColumn.js

```diff
@@ -46,3 +38,11 @@ export const DiskUsageData = (data, id, item) => {
             </Tooltip>
     );
 };
+
+export const displayOS = (data, id, { state }) => {
+    return (
+        state === NO_DATA_STATE ?
+            <span>{ NO_DATA_VALUE }</span> :
+            <span>{ data }</span>
```

Comment:
This is not the case, you can directly display OS info. On backend, it is getting OS info from inventory topic so this information will be there regarding less of rule hit information.
You can handle condition if that value is none.

Reply:
good catch @kgaikwad I updated the condition and checked it against the backend. Thanks!